src/app/components/add-card/add-card.component.css

```diff
@@ -0,0 +1,13 @@
+.date {
+  position: relative;
+  font-family: 'Roboto';
```

Comment:
Let's declare `font-family` globally as this is unlikely to change throughout the app, unless specified otherwise.

Reply:
done!